### DISCUSSION TITLE
[Volar with TS]When prop is hyphenated, the comment of prop is gone

### DISCUSSION THREAD

**Discussion by zouyaoji** [quasarframework/quasar#12508] [General - Components / Directives / etc] [answered]
When written in camel case, comments can come out:
![image](https://user-images.githubusercontent.com/24430116/154521511-5f88603b-1cec-44a7-85a2-26f5f68d291b.png)

but changed it to be hyphenated, it not works.
![image](https://user-images.githubusercontent.com/24430116/154521636-b4611a08-87af-4800-875d-670ffa0a71e2.png)

**Comment by yusufkandemir** [comment, marked as answer]
Please follow this issue: https://github.com/johnsoncodehk/volar/issues/929
Once that is solved, you will get the correct inline docs as you would expect.

**Reply by zouyaoji** [reply]
I got it now.
